modules/earthengine-layers/src/utils.js

```diff
@@ -19,3 +19,16 @@ export function deepEqual(a, b) {
   }
   return true;
 }
+
+// Promisify eeObject methods
+export function promisifyEEObject(eeObject, method, ...args) {
```

Comment:
I'd call this promisifyEEMethod. 

If you wanted to play ion existing idioms and make this more like the built-in Node.js `utils.promisify`, it would return a function that returns a promise, instead of calling that function and returning the promise.

```
const getMapAsync = promisifyEEMethod(eeObject, method);
await getMapAsync();
```